Controllers/ProductsController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using StoresManagement.Data;
+using StoresManagement.Models;
+using StoresManagement.ViewModels;
+
+namespace StoresManagement.Controllers
+{
+    public class ProductsController : Controller
+    {
+        private readonly ApplicationDbContext _context;
+        private readonly IMapper _mapper;
+
+        public ProductsController(ApplicationDbContext context, IMapper mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
+
+        // GET: Products
+        public async Task<IActionResult> Index()
+        {
+            var Products = await _context.Products
```

Comment:
There is widely adopted convention in .net that local variebles names starts with small letter.